src/thunderbots/software/ai/world/field.cpp

```diff
@@ -1,10 +1,13 @@
 #include "field.h"
 
+#include "boost/circular_buffer.hpp"
 #include "geom/rectangle.h"
+#include "util/time/timestamp.h"
 
 Field::Field(double field_length, double field_width, double defense_length,
              double defense_width, double goal_width, double boundary_width,
-             double center_circle_radius)
+             double center_circle_radius, const Timestamp &timestamp,
+             unsigned int buffer_size)
```

Comment:
A default buffer size if probably reasonable here, otherwise we'll have to change the constructor in a few places already in the code.

Reply:
Yup, missing that. I only put it on the field.h definition. Added.

Reply:
Ok NVM, I don't think having the default in the source file works.

Reply:
Yea the default has to be in the header